service/src/main/java/com/codeforcommunity/processor/ProcessorImpl.java

```diff
@@ -24,30 +25,64 @@ public ProcessorImpl(DSLContext db) {
   @Override
   public List<MemberReturn> getAllMembers() {
     List<Member> members = db.selectFrom(Tables.MEMBER).fetchInto(Member.class);
-    return members.stream()
-        .map(member -> new MemberReturn(member.getFirstName(), member.getLastName()))
+    return members.stream().map(member -> new MemberReturn(member.getFirstName(), member.getLastName()))
         .collect(Collectors.toList());
   }
 
+  @Override
+  public boolean attendMeeting(String meetingid, String username) {
+    Result meetingResult = db.fetch("select * from meeting where id=?;", meetingid);
+    // for now assume username is just the first name
+    Result memberResult = db.fetch("select id from member where first_name=?;", username);
+
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.S");
```

Comment:
Parsing the psql TIMESTAMP WITHOUT TIMEZONE into a Java type and comparing it was tough, you should use this exact formatter for this to work